Docs and scripts for running a local demo with two BPA's

ianco opened the pull request:
Demo to run a local universal resolver and 2 BPA's against the BC Gov test ledger (BCovrin test)

See the `./scripts/DEMO-2-BPs.md` document

Uses ianco forks of the universal resolver to start up a resolver against the BCGov leger

etschelp commented:
Well done.  

I have a couple of remarks though. Most important, can you also add the new property bpa.https to backend/business-partner-agent/src/main/resources/application.yml and set it to https?

```
bpa:
  [...]
  name: ${AGENT_NAME:`Business Partner Agent`}
  https: https
```
Like this we do not have to adjust the helm charts, and everything stays backwards compatible.
Generally i would name the property _scheme_ as this is more what you want to do, you want to change the urls scheme. As this means changing a lot of files i leave this up to you. The only blocker that i see is the change in the application.yml

ianco commented:
> Well done.
> 
> I have a couple of remarks though. Most important, can you also add the new property bpa.https to backend/business-partner-agent/src/main/resources/application.yml and set it to https?
> 
> ```
> bpa:
>   [...]
>   name: ${AGENT_NAME:`Business Partner Agent`}
>   https: https
> ```
> 
> Like this we do not have to adjust the helm charts, and everything stays backwards compatible.
> Generally i would name the property _scheme_ as this is more what you want to do, you want to change the urls scheme. As this means changing a lot of files i leave this up to you. The only blocker that i see is the change in the application.yml

@etschelp  I changed the label name to "scheme" and added a default value to the application.yml file

etschelp commented on scripts/.env-example:
```diff
@@ -8,6 +8,7 @@ AGENT_NAME="Business Partner Agent"
 # localhost, ip address, dns name
 BPA_HOST=localhost
 BPA_PORT=8080
+BPA_HTTPS=https
```
Shouldn't this be BPA_SCHEME?

etschelp commented on scripts/DEMO-2-BPs.md:
````diff
@@ -0,0 +1,85 @@
+## Run the first BP Agent
+
+```bash
+git clone https://github.com/ianco/business-partner-agent.git
````
Can you also change this to: https://github.com/hyperledger-labs/business-partner-agent.git?

etschelp commented on scripts/DEMO-2-BPs.md:
````diff
@@ -0,0 +1,85 @@
+```bash
+git clone https://github.com/ianco/business-partner-agent.git
+cd business-partner-agent
+docker build -t ghcr.io/hyperledger-labs/business-partner-agent:local .
````
The build step shouldn't be necessary as the image is being pulled via BPA_DOCKER_IMAGE which is set to the one from GitHub

etschelp commented on scripts/DEMO-2-BPs.md:
````diff
@@ -0,0 +1,85 @@
+First clone and build the sovrin resolver (for bcovrin test):
+
+```bash
+git clone https://github.com/ianco/uni-resolver-driver-did-sov.git
````
Just out of curiosity. Did you do any special changes to the uniresolver?

etschelp commented on scripts/DEMO-2-BPs.md:
````diff
@@ -0,0 +1,85 @@
+In a different shell/directory:
+
+```bash
+git clone https://github.com/ianco/business-partner-agent.git
````
Same here

ianco commented:
@etschelp  updated!
